source/utils/mongodb/mongodb.go

```diff
@@ -0,0 +1,178 @@
+package mongodb
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/mongo/readpref"
+)
+
+var client *mongo.Client
+
+type AprYam struct {
+	ID    primitive.ObjectID `bson:"_id"`
+	Value float64            `bson:"apryam"`
+}
+type AprDegenerative struct {
+	ID    primitive.ObjectID `bson:"_id"`
+	Value map[string]float64 `bson:"aprdegenerative"`
+}
+
+func Connect() {
+	uri := "mongodb://localhost:27017/ugasmedian?readPreference=primary&appname=MongoDB%20Compass&ssl=false"
```

Comment:
that should not be local, you should use the details i sent over setting it on some sort of env file